src/Routes/Repositories/useRepositoryRoutes.tsx

```diff
@@ -53,6 +53,13 @@ export default function useRepositoryRoutes(): TabbedRouteItem[] {
         title: 'Popular repositories',
         route: POPULAR_REPOSITORIES_ROUTE,
         Element: PopularRepositoriesTable,
+        ChildRoutes: [
```

Comment:
in this case on could remove the outer layer here.

hasWrite  ? [
                  { path: DELETE_ROUTE, Element: DeleteContentModal },
                ]
              : []